microsoft/testsuites/mdatp/mdatp_basic.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Please put them under the vm_extensions folder, so we can manage vm extension cases easier. You can have a subfolder there.